ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -561,6 +605,14 @@ export class ObjectDetailComponent implements OnInit {
     }
   }
 
+  checkIfCcColumn(colId: string): boolean {
```

Comment:
As discussed, we use "includes" over here. Might not work in certain substring cases

Reply:
addressed the comment